src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => resolve());
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error()), 3000);
+});
+
+const successHandler = () => {
+  const successMessage = document.createElement('div');
+
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const errorHandler = () => {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1.then(successHandler).catch(errorHandler);
+
+promise2.then(successHandler).catch(errorHandler);
```

Comment:
Both promises use the same success and error handlers. This means that if either promise is rejected, the same error message will be displayed. If you want to differentiate between the two promises in the error messages, consider modifying the `errorHandler` to accept parameters or create separate handlers for each promise.